openc3/lib/openc3/models/scope_model.rb

```diff
@@ -337,8 +339,10 @@ def deploy(gem_path, variables)
       # Scope Cleanup Microservice
       deploy_scopecleanup_microservice(gem_path, variables, @parent)
 
-      # Critical Cmd Microservice
-      deploy_critical_cmd_microservice(gem_path, variables, @parent)
+      if ENTERPRISE
+        # Critical Cmd Microservice
+        deploy_critical_cmd_microservice(gem_path, variables, @parent)
+      end
```

Comment:
Do these new ENTERPRISE conditionals require a migration?

Reply:
Yes removing the critical cmd microservice code requires a migration that only runs on open source.